### PR TITLE
New configure

### PR DESCRIPTION
**Pull request by tacaswell**
replaces #189

### CODE REVIEW

File: tests/test_scaler.py [hunk 1]
```diff
@@ -86,9 +86,11 @@ def test_scaler_functionality(self):
         str(sca)
 
         sca.stage()
-        sca.configure()
+        old, new = sca.configure({})
```
Comment (danielballan) [inline tests/test_scaler.py]: Scaler has a custom `configure` method defined on it which always returns a pair of empty dicts. Gotta scrub out "rogue" ("rouge?") implementations of `configure` and add a non-trivial test.

Comment (danielballan) [inline tests/test_scaler.py]: done here, but Travis is stalling on the scaler tests, so there is probably a typo lurking: https://github.com/danielballan/ophyd/tree/new_configure

Comment (klauer) [inline tests/test_scaler.py]: It may also be a consequence of using a 'fake' scaler in the test, though I haven't really looked into it yet.

**Comment by klauer** [comment]
Looks good, I think.

**Pull request merged**